docker/devel/ubuntu16.04/Dockerfile

```
@@ -0,0 +1,37 @@
+FROM ubuntu:16.04
+
+RUN apt update
+RUN apt install -y software-properties-common
+RUN apt-add-repository ppa:nnstreamer
+RUN apt update
+
+RUN apt install -y git
+RUN git clone https://github.com/nnstreamer/nnstreamer.git /opt/nnstreamer
+
+RUN apt install -y \
+    meson \
+    ninja-build \
+    build-essential \
+    pkg-config \
+    cmake \
+    libglib2.0-dev \
+    libgstreamer1.0-0 \
+    gstreamer1.0-plugins-base \
+    gstreamer1.0-plugins-good \
+    gstreamer1.0-plugins-bad \
+    gstreamer1.0-plugins-ugly \
+    gstreamer1.0-libav \
+    gstreamer1.0-doc \
+    gstreamer1.0-tools \
+    gstreamer1.0-x \
+    gstreamer1.0-alsa \
+    gstreamer1.0-pulseaudio \
+    libgstreamer-plugins-base1.0-dev \
+    libgtest-dev
+
+WORKDIR /opt/nnstreamer/
+RUN meson build
+RUN ninja -C build
+
```

```suggestion
RUN ninja -C build install 
```

Then, you do not need the following lines.